compiler/test-resources/scripting/unglobClasspath.sc

```diff
@@ -1,7 +1,7 @@
-#!bin/scala -classpath 'dist/target/pack/lib/*'
+#!/usr/bin/env -S bin/scala -classpath 'dist/target/pack/lib/*'
 
 // won't compile unless the hashbang line sets classpath
-import org.jline.terminal.Terminal
+import org.jsoup.Jsoup
```

Comment:
for clarification: jline is included in the compiler toolchain by `dist/bin/common`, so the `-classpath 'dist/target/pack/lib/*'` is not necessary. However Jsoup is in the lib directory, making the import break without it.